android/app/src/main/java/com/microsoft/codepush/common/utils/FileUtils.java

```diff
@@ -189,6 +189,7 @@ public static String readFileToString(String filePath) throws IOException {
             Exception e = finalizeResources(
                     Arrays.asList(reader, fileInputStream),
                     "Error closing IO resources when reading file.");
+
```

Comment:
empty line - please add a comment after that or remove it.

Reply:
Removed.